syntax="proto2";
package example;

option cc_generic_services = true;

message myRequest {
      required int32 num1 = 1;
      required int32 num2 = 2;
};

message myResponse {
      required int32 num1 = 1;
      required int32 num2 = 2;
};

service myService {
      rpc game(myRequest) returns (myResponse);
};
